CharlieBackend.Business/Services/AccountService.cs

```diff
@@ -19,11 +19,14 @@ public class AccountService : IAccountService
 
         private readonly IUnitOfWork _unitOfWork;
         private readonly IMapper _mapper;
+        private readonly INotificationService _notification;
 
-        public AccountService(IUnitOfWork unitOfWork, IMapper mapper)
+        public AccountService(IUnitOfWork unitOfWork, IMapper mapper,
+                              INotificationService notification)
```

Comment:
place all parameters on single line, if you choose to break the single line declaration